### PR TITLE
#4457 Application Context - Admin UI

### PR DESCRIPTION
**Pull request by vlt1**
## Description
Mitigates problems described [here](https://github.com/geosolutions-it/MapStore2/issues/4457#issuecomment-557102758)

## Issues
 - #4457 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by tdipisa** [comment]
@offtherailz, @vlt1 for the last point that Lorenzo reported, there is already an [opened issue](https://github.com/geosolutions-it/MapStore2/issues/4535) assigned to @MV88 and in Ready.

The issue about notification involves also other rsource type.

**Comment by offtherailz** [comment]
ok @tdipisa updated the reivew comment

**Comment by vlt1** [comment]
> http://localhost:8081/#/context-creator/11516/general-settings

@offtherailz I get 403 Forbidden

**Comment by offtherailz** [comment]
you should have access no it now. Please tell me when you've done your checks

File: web/client/epics/__tests__/contextmanager-test.js [hunk 1]
```diff
@@ -57,16 +62,18 @@ describe('contextmanager epics', () => {
...
+        const startActions = [contextSaved("Search Text"), loginSuccess(), logout()];
+        testEpic(reloadOnContexts, 3, startActions, (actions) => {
+            actions.forEach(a => {
+                expect(a.type).toBe(SEARCH_CONTEXTS);
+                expect(a.options).toExist();
+                expect(a.options.params).toExist();
+                expect(a.options.params.start).toBe(0);
+                expect(a.options.params.limit).toBe(12);
+                expect(a.text).toBe("test");
+                done();
```
Comment (offtherailz) [inline web/client/epics/__tests__/contextmanager-test.js]: This test stops at first iteration calling done. But you're expecting 3 actions.Maybe you have to test event the second and the 3rd action

**Comment by tdipisa** [comment]
@vlt1 I completelly agree with @offtherailz about the following

>>If, as I suppose, on logout you have to be redirected to the home page, you can simply clean-up unnecessary code and add to the page the "Redirect" plugin that should do the work for you (I'm not sure if need to be configured or what, please take a look at how it's used in admin interface).

Honestly I missed this point when I looked at the issue description the first time, please @francoperversi pay attention you too. On logout a redirect to the home page is needed in that case, anyway if there are some wrong behaviors in refreshing the contexts list in some other cases they must be fixed.

**Pull request merged**